src/models/spaces/cell-zoom/cell-zoom-row.ts

```diff
@@ -0,0 +1,68 @@
+import { types, Instance } from "mobx-state-tree";
+import { ChartDataModelType, ChartDataModel } from "../charts/chart-data";
+import { DataPoint, ChartDataSetModel, ChartColors, DataPointType } from "../charts/chart-data-set";
+import { CellMouseModel } from "../../../components/spaces/cell-zoom/cell-mouse";
+import { kSubstanceNames } from "./cell-zoom";
+
+export const Organelle = types.enumeration("type", [
```

Comment:
I moved a couple of enumerations into this file, even though I'd prefer to keep them all at the top level `CellZoomModel`. If I didn't move them, MST wasn't picking them up properly somehow, even when they were imported...perhaps one of you could help me figure out what's going on there?

Reply:
Been playing around with it and I can't work it out either. It seems like importing `Organelle` from another file should be exactly the same was when `cell-mouse.ts` imports `MouseColor`:

```
export const CellMouseModel = types
  .model("CellMouse", {
    id: types.optional(types.identifier, () => uuid()),
    baseColor: MouseColor
  })
```

where `MouseColor` is defined elsewhere as 

```
export const MouseColor = types.enumeration("type", ["brown", "white", "tan"]);
```

I'm just not seeing the difference between these approaches.